File: abc/179/b.c
#include <stdio.h>
#include <string.h>

int main(){
  // input
  int N; scanf("%d", &N);
  int i, j, D[N][2];
  for(i = 0; i < N; ++i){
    for(j = 0; j < 2; ++j){
      scanf("%d", &D[i][j]);
    }
  }
  
  // judge
  int flag = 0;
  for(i = 0; i < N-2; ++i){
    if(D[i][0]==D[i][1] && D[i+1][0]==D[i+1][1] && D[i+2][0]==D[i+2][1]){
      flag = 1;
    }
  }

  // output
  if(flag){
    printf("Yes\n");
  }
  else{
    printf("No\n");
  }

  return 0;
}

File: ABC/179/b.py
N = int(input())
D = []
for i in range(N):
  D.append(list(map(int, input().split())))

flag = False
for i in range(N-2):
  if D[i][0]==D[i][1] and D[i+1][0]==D[i+1][1] and D[i+2][0]==D[i+2][1]:
    flag = True
if flag:
  print('Yes')
else:
  print('No')